examples/using-marking-definitions.json

```diff
@@ -0,0 +1,58 @@
+{
+  "type": "bundle",
+  "id": "bundle--b56c1e2e-a40c-44ca-83dd-09e25936d273",
+  "spec_version": "2.0",
+  "objects": [
+    {
+      "type": "identity",
+      "id": "identity--611d9d41-dba5-4e13-9b29-e22488058ffc",
+      "created": "2017-04-14T13:07:49.812Z",
+      "modified": "2017-04-14T13:07:49.812Z",
+      "name": "Stark Industries",
+      "identity_class": "organization",
+      "contact_information": "info@stark.com",
+      "sectors": [
+        "defense"
+      ],
```

Comment:
Delete this comma to make the JSON well-formed. I think this is what caused the Travis error.